From 4a890d7e173678644882e6b863f3650e30d33052 Mon Sep 17 00:00:00 2001
From: Srinivas Pandruvada <srinivas.pandruvada@intel.com>
Date: Thu, 18 Feb 2016 11:33:50 -0800
Subject: [PATCH] Regression for default config file

f99f2b59fbbca04a13cad3f7d2dbc985bc7ee0cd caused regression
where for Android the default path is changed to TDRUNDIR.
This will cause issue to upgrade thermald.

Thanks to Bruno Pagani "ArchangeGabriel" to identify and root
causing this.
---
 src/thd_parse.cpp | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/src/thd_parse.cpp b/src/thd_parse.cpp
index 7e8e84c..f73d43e 100644
--- a/src/thd_parse.cpp
+++ b/src/thd_parse.cpp
@@ -60,8 +60,13 @@ char *cthd_parse::char_trim(char *str) {
 
 cthd_parse::cthd_parse() :
 		matched_thermal_info_index(-1), doc(NULL), root_element(NULL) {
+	std::string name_conf = TDCONFDIR;
 	std::string name_run = TDRUNDIR;
+#ifdef ANDROID
 	filename = name_run + "/" + "thermal-conf.xml";
+#else
+	filename = name_conf + "/" + "thermal-conf.xml";
+#endif
 	filename_auto = name_run + "/" + "thermal-conf.xml.auto";
 }
 
-- 
2.8.1

